controllers/progressivesync_controller_test.go

```diff
@@ -88,6 +88,10 @@ var _ = Describe("ProgressiveRollout Controller", func() {
 				ObjectMeta: metav1.ObjectMeta{
 					Name:      "app",
 					Namespace: localNS,
+					Annotations: map[string]string{
```

Comment:
Out of curiosity, why are we adding those annotations? As default you can have none.

Reply:
I think initialising the empty map didn't work and was erroring with
>assignment to entry in nil map

However, maybe I didn't initialise it correctly 🤔

Reply:
This should now be fixed and the initial map should be empty.